pyseir/inference/model_fitter.py

```diff
@@ -182,6 +152,46 @@ def __init__(self,
         self.dof_cases = None
         self.dof_hosp = None
 
+    def set_inference_parameters(self):
+        """
+        Setup inference parameters based on data availability and manual
+        overrides.  As data becomes more sparse, we further constrain the fit,
+        which improves stability substantially.
+        """
+        self.fit_params = self.DEFAULT_FIT_PARAMS
+        # Update any state specific params.
+        for k, v in self.PARAM_SETS.items():
+            if self.state_obj.abbr in k:
+                self.fit_params.update(v)
+
+        self.fit_params['fix_hosp_fraction'] = self.hospitalizations is None
+        if self.hospitalizations is None:
+            self.fit_params['hosp_fraction'] = 1
+
+        if len(self.fips) == 5:
+            t0_guess = list(self.times)[np.argwhere(np.cumsum(self.observed_new_cases) >= 3)[0][0]]
+
+            state_fit_result = load_inference_result(fips=self.state_obj.fips)
+
+            self.fit_params['t0'] = t0_guess # state_fit_result['t0']
+
+            total_cases = np.sum(self.observed_new_cases)
+            self.fit_params['log10_I_initial'] = np.log10(3 / self.fit_params['test_fraction'])
```

Comment:
Instead of 3, should this be cumulative observed new cases at t0_guess?

Reply:
I'll add a comment. The problem is that this is e.g. 20x observed cases, which is fairly late in the simulation, so I found it much better to fit using these parameters after a lot of trial and error.